### PR TITLE
Removing Forever dependency

### PR DESCRIPTION
**Pull request by codingtwinky**
I will try to refactor it to es6 without async.

### CODE REVIEW

File: bin/ungit [hunk 1]
```diff
@@ -11,19 +10,10 @@ var BugTracker = require('../src/bugtracker');
...
-  env: { LANG: 'en_US.UTF-8' }
-});
-
-process.on('exit', function() {
-  child.kill();
-  console.log('Exit: killed children')
-});
+var child = child_process.spawn('node',
+  [path.join(__dirname, '..', 'src', 'server.js')].concat(process.argv.slice(2)),
+  { cwd: path.join(process.cwd(), '..') }
```
Comment (campersau) [inline bin/ungit]: What about the `env` setting?

File: bin/ungit [hunk 2]
```diff
@@ -11,19 +10,10 @@ var BugTracker = require('../src/bugtracker');
 var bugtracker = new BugTracker('launcher');
 var usageStatistics = require('../src/usage-statistics');
 
-var child = new (forever.Monitor)(path.join(__dirname, '..', 'src', 'server.js'), {
-  silent: false,
-  minUptime: 2000,
-  max: config.maxNAutoRestartOnCrash,
```
Comment (campersau) [inline bin/ungit]: Is `maxNAutoRestartOnCrash` used anywhere else?

File: bin/ungit (hunk 1 above)
Comment (codingtwinky) [reply bin/ungit]: It didn't seems to be respected as apparent by #959.  I may leave it in with a note though...

File: bin/ungit (hunk 2 above)
Comment (codingtwinky) [reply bin/ungit]: Not really...  From my tests forever monitor actually never kicked in and process didn't die during errors.  Errors seems to be captured and handled so..

Comment (campersau) [reply bin/ungit]: Ok, so that means the config setting can be removed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,7 +3,9 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 Use the following format for additions: ` - VERSION: [feature/patch (if applicable)] Short description of change. Links to relevant issues/PRs.`
 
-- 1.1.31: Bump dependencies
+- 1.1.31:
```
Comment (campersau) [inline CHANGELOG.md]: Also bump version in `package.json`.

**Pull request merged**